Maestro/test/Microsoft.DotNet.Maestro.SubscriptionValidation/packages.config

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="xunit" version="2.1.0" targetFramework="net452" />
+  <package id="xunit.abstractions" version="2.0.0" targetFramework="net452" />
+  <package id="xunit.assert" version="2.1.0" targetFramework="net452" />
+  <package id="xunit.core" version="2.1.0" targetFramework="net452" />
+  <package id="xunit.extensibility.core" version="2.1.0" targetFramework="net452" />
+  <package id="xunit.extensibility.execution" version="2.1.0" targetFramework="net452" />
+  <package id="xunit.runner.msbuild" version="2.1.0" targetFramework="net461" />
```

Comment:
Maybe it's just my config, but `xunit.runner.visualstudio` means I can see the tests in the test explorer, and otherwise I can't. Running during the build is fine with me, though.